modules/logging-loki-storage-azure.adoc

```diff
@@ -24,3 +24,26 @@ $ oc create secret generic logging-loki-azure \
   --from-literal=account_key="<azure_account_key>"
 ----
 <1> Supported environment values are `AzureGlobal`, `AzureChinaCloud`, `AzureGermanCloud`, or `AzureUSGovernment`.
+
+== Azure storage for STS enabled clusters
+
+If your cluster has STS enabled, the Cloud Credential Operator (CCO) supports short-term authentication using Azure AD Workload Identity.
+
+.Required data:
+environment:: Azure Storage account environment.
+container:: Azure Storage account container
+account_name::  Azure Storage account name
+client_id:: Azure Workload Identity's Client ID
+tenant_id:: Azure Account's Tenant ID holding the workload identity for LokiStack
+subscription_id:: Azure Account's Subscription ID holding the workload identity for LokiStack
+`Storage Blob Data Contributor`:: Azure role required. 
+
+You can create the Loki object storage secret manually by running the following command:
+
+[source,terminal,subs="+quotes"]
+----
+oc -n openshift-logging create secret generic logging-loki-azure
+--from-literal=environment="<azure_environment>"
+--from-literal=account_name="<storage_account_name>"
+--from-literal=container="<container_name>"
```

Comment:
A little typo on these since it is multiline. it should include '\' after each line

oc -n openshift-logging create secret generic logging-loki-azure \
--from-literal=environment="<azure_environment>" \
--from-literal=account_name="<storage_account_name>" \
--from-literal=container="<container_name>"